.travis.yml

```diff
@@ -11,13 +11,13 @@ matrix:
   include:
     - python: 2.7
       env: TOXENV=py27 DDB=true CODECOV=true
-    - python: pypy-5.7.1
+    - python: pypy
```

Comment:
Which version does this peg us to?

Reply:
This bumps us to the latest they have, 5.8.0. I'm hoping this automatically bumps us to 5.9.0 when they add it